.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish SDK
+on:
+  push:
+    branches:
+      - release
+
+jobs:
+  publish:
+    runs-on: ubuntu-18.04
+    steps:
+      # Get code into container
+      - name: Checkout code
+        uses: actions/checkout@v2
+      # Set up Node 12
+      - name: Install Node v12
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
+          registry-url: "https://registry.npmjs.org"
+      # Install app
+      - name: Install application
+        run: npm install
+      # Publish app
+      - name: Publish application
+        run: npm publish
```

Comment:
`npm publish` tags the package as `latest` if no tag is specified, we need to control the version tag